checkout.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Checkout</title>
+    <link rel="stylesheet" href="./styles/master.css">
+    <style>
+      body {
+         background-color: lightgray;
+      }
+    </style>
+  </head>
+
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" width="200" alt="adoptadog logo">
+      <p>Seattle's premiere dog adoption agency</p>
+
+      <img src="images/cart-icon.svg" width="24" alt="shopping cart icon">
+      <p>$1234.56</p>
+      <button type="button">Checkout</button>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main>
+      <h2>Checkout</h2>
+
+
+      <form class="" action="checkout.html" method="post">
+        <label for="name">Name</label><br>
+        <input type="text" id="name" name="name"><br>
+
+        <label for="email">Email Address</label><br>
+        <input type="email" id="email" name="email"><br>
+
+        <label for="street">Street Address</label><br>
+        <input type="text" id="street" name="street"><br>
+
+        <label for="city">City</label><br>
+        <input type="text" id="city" name="city"><br>
+
+        <label for="state">State</label><br>
+        <input type="text" id="state" name="state"><br>
+
+        <label for="zip_code">Zip Code</label><br>
```

Comment:
nice use of for's and id's. These <br>'s should be necessary. You can use display properties (block, inline-block, inline, etc) to format these out.